### PR TITLE
fix: envoy statistics config documentation

### PR DESCRIPTION
**Pull request by chinaran**
refer: https://www.envoyproxy.io/docs/envoy/latest/configuration/upstream/cluster_manager/cluster_stats
associate to: https://github.com/istio/api/pull/2592

1. Istio uses `outlier_detection` for `circuit_breakers`
2. There are 6 retry metrics (`cluster.<name>.upstream_rq_retry*`) 
3. There are many upstream connections metrics (`cluster.<name>.upstream_cx_*`) 
4. Added `upstream_rq_timeout` suffixed example

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [X] Configuration Infrastructure
- [ ] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/en/docs/reference/config/istio.mesh.v1alpha1/index.html [hunk 1]
```diff
@@ -2875,14 +2875,15 @@ <h2 id="ProxyConfig">ProxyConfig</h2>
 (<code>sidecar.istio.io/statsInclusionPrefixes</code>,
 <code>sidecar.istio.io/statsInclusionRegexps</code>, and
 <code>sidecar.istio.io/statsInclusionSuffixes</code>). For example, to enable stats
-for circuit breaker, retry, and upstream connections, you can specify stats
+for circuit breaker, retry, upstream connections and request timeout, you can specify stats
```
Comment (ericvn) [inline content/en/docs/reference/config/istio.mesh.v1alpha1/index.html:2878]: These changes need to be made in the istio/api repo where this file is copied from: https://github.com/istio/api/blob/master/mesh/v1alpha1/istio.mesh.v1alpha1.pb.html#L2876 which is generated from https://github.com/istio/api/blob/master/mesh/v1alpha1/proxy.proto#L561.

File: content/en/docs/reference/config/istio.mesh.v1alpha1/index.html [hunk 2]
```diff
@@ -2875,14 +2875,15 @@ <h2 id="ProxyConfig">ProxyConfig</h2>
...
   inclusionRegexps:
-    - .*circuit_breakers.*
-  inclusionPrefixes:
-    - upstream_rq_retry
-    - upstream_cx
+    - .*outlier_detection.*
+    - .*upstream_rq_retry.*
+    - .*upstream_cx_.*
+  inclusionSuffixes:
+    - upstream_rq_timeout
```
Comment (ericvn) [inline content/en/docs/reference/config/istio.mesh.v1alpha1/index.html:2886]: Changes in istio/api.

File: content/zh/docs/reference/config/istio.mesh.v1alpha1/index.html [hunk 3]
```diff
@@ -2875,14 +2875,15 @@ <h2 id="ProxyConfig">ProxyConfig</h2>
 (<code>sidecar.istio.io/statsInclusionPrefixes</code>,
 <code>sidecar.istio.io/statsInclusionRegexps</code>, and
 <code>sidecar.istio.io/statsInclusionSuffixes</code>). For example, to enable stats
-for circuit breaker, retry, and upstream connections, you can specify stats
+for circuit breaker, retry, upstream connections and request timeout, you can specify stats
```
Comment (ericvn) [inline content/zh/docs/reference/config/istio.mesh.v1alpha1/index.html:2878]: This file is the same as the English file and copied from istio/api.

File: content/en/docs/ops/configuration/telemetry/envoy-stats/index.md [hunk 4]
```diff
@@ -63,10 +63,11 @@ spec:
...
         inclusionRegexps:
-          - ".*circuit_breakers.*"
-        inclusionPrefixes:
-          - "upstream_rq_retry"
-          - "upstream_cx"
+          - ".*outlier_detection.*"
+          - ".*upstream_rq_retry.*"
+          - ".*upstream_cx_.*"
+        inclusionSuffixes:
+          - "upstream_rq_timeout"
```
Comment (ericvn) [inline content/en/docs/ops/configuration/telemetry/envoy-stats/index.md:70]: Looking for input from @istio/wg-policies-and-telemetry-maintainers

File: content/en/docs/reference/config/istio.mesh.v1alpha1/index.html (hunk 1 above)
Comment (chinaran) [reply content/en/docs/reference/config/istio.mesh.v1alpha1/index.html]: please see: https://github.com/istio/api/pull/2592

File: content/zh/docs/reference/config/istio.mesh.v1alpha1/index.html (hunk 3 above)
Comment (chinaran) [reply content/zh/docs/reference/config/istio.mesh.v1alpha1/index.html]: please see: https://github.com/istio/api/pull/2592

File: content/en/docs/ops/configuration/telemetry/envoy-stats/index.md (hunk 4 above)
Comment (lei-tang) [reply content/en/docs/ops/configuration/telemetry/envoy-stats/index.md:70]: https://github.com/istio/api/pull/2592 has not been merged yet. I suggest holding off updating the doc updates until the API changes and the implementation have been merged.

Comment (ericvn) [reply content/en/docs/ops/configuration/telemetry/envoy-stats/index.md:70]: The doc updates will happen when the istio/api PR merges as there is a nightly job which will scrape them into preliminary.istio.io.

Comment (chinaran) [reply content/en/docs/ops/configuration/telemetry/envoy-stats/index.md:70]: https://github.com/istio/api/pull/2592 has been merged.

Comment (zirain) [reply content/en/docs/ops/configuration/telemetry/envoy-stats/index.md:70]: please wait https://github.com/istio/istio.io/pull/12383, then rebase from master, thanks~

**Comment by ericvn** [comment]
I'll wait for approval first from @istio/wg-docs-maintainers-chinese for the translated file, otherwise my approval will merge with changes.

File: content/zh/docs/ops/configuration/telemetry/envoy-stats/index.md [hunk 5]
```diff
@@ -45,7 +45,7 @@ Envoy 只收集在 `stats_matcher` JSON 字段中能匹配上 `inclusion_list` 
 如果您基于 Envoy 建立或者维护仪表盘或者告警，**强烈建议**您在**升级 Istio 之前**先在[金丝雀环境](/zh/docs/setup/upgrade/canary/index.md)检查统计信息。
 {{< /tip >}}
 
-想让 Istio 代理能够记录更多的统计信息，您可以在您的网格配置中添加 [`ProxyConfig.ProxyStatsMatcher`](/zh/docs/reference/config/istio.mesh.v1alpha1/#ProxyStatsMatcher)。例如，为了启用断路器、重试、全局上游连接的统计数据，您可以指定如下的数据统计的匹配配置：
+想让 Istio 代理能够记录更多的统计信息，您可以在您的网格配置中添加 [`ProxyConfig.ProxyStatsMatcher`](/zh/docs/reference/config/istio.mesh.v1alpha1/#ProxyStatsMatcher)。例如，为了启用断路器、请求重试、全局上游连接和请求超时的统计数据，您可以指定如下的数据统计的匹配配置：
```
Comment (windsonsea) [inline content/zh/docs/ops/configuration/telemetry/envoy-stats/index.md:48]:
```suggestion
想让 Istio 代理能够记录更多的统计信息，您可以在您的网格配置中添加 [`ProxyConfig.ProxyStatsMatcher`](/zh/docs/reference/config/istio.mesh.v1alpha1/#ProxyStatsMatcher)。例如，为了全局启用断路器、请求重试、上游连接和请求超时的统计数据，您可以指定如下的数据统计的匹配配置：
```
Seems `globally` is for the whole clause, right?

**Review by zirain — APPROVED** [review]
for `zh` part

File: content/en/docs/ops/configuration/telemetry/envoy-stats/index.md (hunk 4 above)
Comment (chinaran) [reply content/en/docs/ops/configuration/telemetry/envoy-stats/index.md:70]: @zirain already rebase.

**Pull request merged**